src/expressions/operator-expr.md

```diff
@@ -1,5 +1,18 @@
 # Operator expressions
 
+> **<sup>Syntax</sup>**  
+> _OperatorExpression_ :  
```

Comment:
Missing GroupedExpression.

Reply:
Grouped expression will be located directly on the _Expression_ production. I think it would be out of place if put together with the OperatorExpressions -- even though it is in the same section. I put it in the operator expression section mainly because I think it is too small to be on a separate section by itself.

What do you think?

Reply:
I think grammar should be associated with its prose. GroupedExpression is on this page, so it should show up here. If it feels out of place in the grammar, it should feel out of place on the prose and moved to its own page, albeit the shortness is concerning. Although some examples could help with that.